src/services/WolfDen.API/Controllers/Attendence/AttendenceLog.cs

```diff
@@ -0,0 +1,61 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using QuestPDF.Fluent;
+using WolfDen.Application.Requests.Commands.Attendence.StatusUpdation;
+using WolfDen.Application.Requests.Queries.Attendence.DailyStatus;
+
+
+namespace WolfDen.API.Controllers.Attendence
+{
+    [Route("api/attendance")]
+    [ApiController]
+    public class AttendenceLog : ControllerBase
+    {
+        private readonly IMediator _mediator;
+        private readonly PdfService _pdfService;
+      
+        public AttendenceLog(IMediator mediator,PdfService pdfService)
+        {
+            _mediator = mediator;
+            _pdfService = pdfService;  
+        }
+
+        [HttpPost("{employeeId}/status")]

```

Comment:
httppatch

Reply:
patch is used when is not updating the full row right?,but here the entry is created in the table with this httpost,was not an updation.